rfcs/0005-pablo-distribution-assets/0005-pablo-distribution.adoc

```diff
@@ -475,22 +475,17 @@ note right: as per section 5.3.1.5
 stop
 ----
 
-==== Claim/Unstake
+==== Unstake
 
-[plantuml,images/claim,png]
+[plantuml,images/unstake,png]
 ----
 start
 ->assetId\npositionId;
 :rewards_pool = readStorage(assetId);
 :position = readStorage(positionId);
 :penalty = reward_pool.lock.unlock_penalty;
```

Comment:
With this change we are avoiding the penalty entirely. Let's keep this algo same as before. Important thing is to show the entire algorithm in a simple way so no need for it to reuse other algos :)

Reply:
6335363da6fb63a1471a6081e893998fb28a7a9a